ale_linters/go/gometalinter.vim

```diff
@@ -0,0 +1,61 @@
+" Author: Ben Reedy <https://github.com/breed808>
+" Description: Adds support for the gometalinter suite for Go files
+
+" Disable gometalinter by default
+if !exists('g:ale_go_gometalinter_enabled')
+    let g:ale_go_gometalinter_enabled = 0
+endif
+
+if !exists('g:ale_go_gometalinter_options')
+    let g:ale_go_gometalinter_options = ''
+endif
+
+function! ale_linters#go#gometalinter#GetCommand(buffer) abort
+    if g:ale_go_gometalinter_enabled == 1
```

Comment:
Instead of doing this, you should edit the default list of enabled linters for `Go`. Look for `s:default_ale_linters` in `autoload/ale/linter.vim`. That will update the output of `:ALEInfo` appropriately. (It will show up in the list of available linters, but not in the enabled list.)

Write a paragraph about this at the top of the Go integration file. Look at the Rust file for an example of some documentation like that.